src/Humanizer.Tests/ApiApprover/PublicApiApprovalTest.approve_public_api.approved.txt

```diff
@@ -245,6 +245,10 @@ public class In
 
 public class InflectorExtensions
 {
+    public void AddIrregular(string singular, string plural) { }
```

Comment:
I think something gone wrong with the rebase, as this methods are part of another PR.